src/controllers/users.js

```diff
@@ -1,24 +1,25 @@
-/* eslint-disable linebreak-style */
-import { Sequelize, sequelize } from '../models/index';
+import userHelpers from '../utils/userHelperMethods';
 
-const Users = require('../models/User')(sequelize, Sequelize.DataTypes);
+class UserControllers{
+    static async signupController(req, res) {
+        try{
```

Comment:
Validations are handled under the Validation PR. Please do not bother.
You could, however, review the [Validation PULL REQUEST](https://github.com/andela/rogue-backend/pull/26)and make your suggestions.

Reply:
We agreed on using sequelize's in-built validation to also catch error in the database too